modules/api/src/main/scala/cards/nine/api/NineCardsDirectives.scala

```diff
@@ -54,46 +53,59 @@ class NineCardsDirectives(
   with PathDirectives
   with RouteDirectives
   with SecurityDirectives
+  with FutureDirectives
   with JsonFormats {
 
   import cards.nine.api.accounts.JsonFormats._
 
-  implicit def fromTaskAuth[T](auth: ⇒ Task[Authentication[T]]): AuthMagnet[T] =
-    new AuthMagnet(onSuccess(auth))
+  implicit def fromTaskAuth[T](auth: ⇒ Task[AuthenticationResult[T]]): Future[AuthenticationResult[T]] =
+    auth.unsafePerformAsyncFuture()
+
+  val nineCardChallenge = HttpChallenge("NineCard", None)
```

Comment:
Temporarily added because akka-http requires `HttpChallenge` for `AuthenticationFailedRejection` and it's probably not the best place for this.